frontend/templates/express/src/app.js.hbs

```diff
@@ -0,0 +1,120 @@
+require("express");
```

Comment:
This PR should have some documentation about how the generated system works, along the lines of https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/main/app/README.md